src/libQuiet/createTranslator.js

```diff
@@ -0,0 +1,30 @@
+import libQuietProvider from './libQuietProvider';
+
+export default function createTranslator(type, profile, sampleRate) {
+    const libQuiet = libQuietProvider.get();
+
+    const allProfiles
+        = libQuiet.intArrayFromString(JSON.stringify({ profile }));
+    const selectedProfileKey = libQuiet.intArrayFromString('profile');
+
+    const translatorProfile = libQuiet.ccall(
+        `quiet_${type}_profile_str`,
+        'pointer',
+        [ 'array', 'array' ],
+        [ allProfiles, selectedProfileKey ]
+    );
+
+    // libquiet internally works at 44.1kHz but the local sound card may be
```

Comment:
Probably worth fixing up this comment, since it just applies to the sampleRate arg, but really we're creating the encoder/decoder here :)